components/steplist/package.json

```diff
@@ -19,13 +19,13 @@
   },
   "peerDependencies": {
     "@spectrum-css/icon": ">=3",
-    "@spectrum-css/vars": ">=9"
+    "@spectrum-css/tokens": ">=11"
   },
   "devDependencies": {
-    "@spectrum-css/component-builder": "^4.0.14",
+    "@spectrum-css/component-builder-simple": "^2.0.16",
     "@spectrum-css/icon": "^4.0.1",
-    "@spectrum-css/tooltip": "^5.1.10",
-    "@spectrum-css/vars": "^9.0.8",
+    "@spectrum-css/tokens": "^11.2.0",
+    "@spectrum-css/tooltip": "^5.1.8",
```

Comment:
is there any reason you are now referencing `"@spectrum-css/tooltip": "^5.1.10"` instead of `5.1.8`

Reply:
thnx for noticing this. it was a mistake. fixed it!